dmsg-daemon/cmd/internal/api/api.go

```diff
@@ -0,0 +1,33 @@
+package api
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/go-chi/chi"
+	"github.com/sirupsen/logrus"
+	"github.com/skycoin/dmsg/httputil"
+)
+
+type API struct {
+	http.Handler
+	startedAt time.Time
```

Comment:
Not sure this will be needed. This will run locally. We will not monitor this most likely.